library/core/class.format.php

```diff
@@ -2001,29 +1430,20 @@ public static function serialize($mixed) {
      * Takes a mixed variable, formats it for display on the screen as plain text.
      *
      * @param mixed $mixed An object, array, or string to be formatted.
+     * @param bool|null $addBreaks
      * @return string Sanitized HTML.
+     * @deprecated Formats\TextFormat::renderHtml()
      */
-    public static function text($mixed, $addBreaks = true) {
+    public static function text($mixed, $addBreaks = null) {
         if (!is_string($mixed)) {
```

Comment:
This refactoring does not look the same as previous implementation.

Reply:
It does not see the note in the top of the PR. This parameter is deprecated (deprecation notice just below).

After some extensive discussion with @tburry It was determine that this function is being misused anyways, and that usages attempting should just be replaced with `htmlspecialchars`.